applications/MeshingApplication/custom_processes/nodal_values_interpolation_process.h

```diff
@@ -154,7 +154,7 @@ class NodalValuesInterpolationProcess
             Vector ShapeFunctions;
             Element::Pointer pElement;
             
-            const bool IsFound = PointLocator.FindPointOnMeshSimplified(itNode->Coordinates(), ShapeFunctions, pElement, mMaxNumberOfResults);
+            const bool IsFound = PointLocator.FindPointOnMeshSimplified(itNode->Coordinates(), ShapeFunctions, pElement, mMaxNumberOfResults, 5.0e-2);
```

Comment:
This is a very large tolerance! Is it correct?

Reply:
Indeed, but didn't work otherwise

Reply:
It look like you have some "algorithmic" problem hidden behind....

Reply:
Could be, I just know that the problem is solved modifying the tolerance. This problem already happened in the past

Reply:
Can u see if by using a small tolerance it works? If it does not we shall see why it doesnt

Reply:
I tried with all the intermediate tolerances, even 1.0e-3 didn't work

Reply:
It would be work to see what is failing...

Reply:
:+1: